draft-packages/stories/Modal.stories.tsx

```diff
@@ -677,3 +683,29 @@ export const NestedModal = () => (
 NestedModal.story = {
   name: "Nested confirmation modal",
 }
+
+export const Roadblock = () => (
+  <ModalStateContainer isInitiallyOpen={false}>
+    {({ open, close, isOpen }) => (
+      <div>
+        <Button label="Open modal" onClick={open} />
+        <RoadblockModal
+          isOpen={isOpen}
+          title="Roadblock title"
+          onDismiss={close}
+        >
+          <div style={{ textAlign: "center" }}>
+            <Paragraph variant="body">
+              Tell users why they're unable to proceed and what needs to happen
+              first.
+            </Paragraph>
+          </div>
+        </RoadblockModal>
+      </div>
+    )}
+  </ModalStateContainer>
+)
+
+Roadblock.story = {
+  name: "Roadblock",
```

Comment:
Calling this story `Confirmation (roadblock)` will put it next to the other confirmation modals and will be a little nicer for discoverability for this component. 

![Screen Shot 2020-10-01 at 3 47 01 pm](https://user-images.githubusercontent.com/1621182/94772628-5e61a980-03fd-11eb-9fc8-a5c14f04e50c.png)


```suggestion
  name: "Confirmation (roadblock)",
```

Reply:
Technically it's not a Confirmation modal (hence why it's not just a variant of the existing Confirmation modals). Do you think labelling it as a Confirmation modal in storybook is still a good idea?

Reply:
Right you are! My mistake - what you have now is consistent with the UI kit so no change required.